packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/TimelineHeaderRow/TimelineCollapser.css

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright (c) 2017 Uber Technologies, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.TimelineCollapser {
+  float: right;
+  margin: 0 0.8rem 0 0;
+  display: inline-block;
+}
+
+.TimelineCollapserBtn {
+  margin-right: 0.3rem;
+}
+
+.ExpandBtn {
+  transform: rotate(90deg);
+}
```

Comment:
The `ExpandBtn` class should be scoped to `TimelineCollapser` to prevent clashes with other styles.

Also, if written in the following way both classes don't need to be added to the component:

```css
.TimelineCollapser--btn,
.TimelineCollapser--btn-expand {
  margin-right: 0.3rem;
}

.TimelineCollapser--btn-expand {
  transform: rotate(90deg);
}
```